src/main/java/urlshortener/service/ShortURLService.java

```diff
@@ -38,4 +38,8 @@ public ShortURL save(String url, String sponsor, String ip) {
             .build();
     return shortURLRepository.save(su);
   }
+
+  public Long getTotalURL() {
+    return shortURLRepository.count();
+  } // NOSONAR
```

Comment:
Why is it necessary?

Reply:
![image](https://user-images.githubusercontent.com/45603226/99911653-35231280-2cf6-11eb-96ee-300f5f1506b1.png)
Sorry I thought the whole function was covered by the comment